rendermodules/dataimport/schemas.py

```diff
@@ -154,3 +154,72 @@ class GenerateEMTileSpecsParameters(RenderParameters):
 class GenerateEMTileSpecsOutput(DefaultSchema):
     stack = Str(required=True,
                 description="stack to which generated tiles were added")
+
+
+class MakeMontageScapeSectionStackParameters(RenderParameters):
+    montage_stack = mm.fields.Str(
+        required=True,
+        metadata={'description':'stack to make a downsample version of'})
```

Comment:
this works.. but description = " .... " is more consistent with the other schemas... might help keep this uniform if we move in another direction in the future.